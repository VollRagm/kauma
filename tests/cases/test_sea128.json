{
    "testcases": {
        "b856d760-023d-4b00-bad2-15d2b6da22fe": {
            "action": "sea128",
            "arguments": {
                "mode": "encrypt",
                "key": "istDASeincoolerKEYrofg==",
                "input": "yv66vvrO263eyviIiDNEVQ=="
            }
        },
        "254eaee7-05fd-4e0d-8292-9b658a852245": {
            "action": "sea128",
            "arguments": {
                "mode": "decrypt",
                "key": "istDASeincoolerKEYrofg==",
                "input": "D5FDo3iVBoBN9gVi9/MSKQ=="
            }
        }
    }
}