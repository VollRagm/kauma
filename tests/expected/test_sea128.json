{
    "responses": {
        "b856d760-023d-4b00-bad2-15d2b6da22fe": {
            "output": "D5FDo3iVBoBN9gVi9/MSKQ=="
        },
        "254eaee7-05fd-4e0d-8292-9b658a852245": {
            "output": "yv66vvrO263eyviIiDNEVQ=="
        }
    }
}